src/library/Uno.Material/Styles/Controls/v2/RatingControl.xaml

```diff
@@ -0,0 +1,270 @@
+                            <StackPanel x:Name="RatingBackgroundStackPanel"
+                                        Margin="20,20,0,20"
+                                        Background="Transparent"
+                                        Orientation="Horizontal" />
+                            <TextBlock x:Name="Caption"
+									   Height="{ThemeResource SecondaryRatingControlCaptionHeight}"
+                                       Margin="4,9,20,0"
+                                       VerticalAlignment="Center"
+                                       AutomationProperties.AccessibilityView="Raw"
+                                       AutomationProperties.Name="RatingCaption"
+                                       Foreground="{ThemeResource SecondaryRatingControlCaptionForeground}"
+                                       FontFamily="{ThemeResource SecondaryRatingControlCaptionFontFamily}"
+                                       IsHitTestVisible="False"
+                                       Style="{ThemeResource SecondaryRatingControlCaptionStyle}"
+                                       FontWeight="Bold"
+                                       Text="{TemplateBinding Caption}"
+                                       TextLineBounds="TrimToBaseline" />
+                            <!-- 4 = 8 item spacing +4 of magic redline spacing -8 to compensate for scale of the last RatingItem -->
+                            <!--
+                                NB: The redlines say 8px, but it's really 12 px because:
+                                Designer note: The value between the last glyph and first text character is 12px.
+                                (There's 4px of whitespace accounted for in the text area in the redline)
+                            -->
+                            <!-- TODO MSFT: 9925444 Fix vertical alignment in all text scenarios -->
+                        </StackPanel>
+
+                        <ContentPresenter x:Name="ForegroundContentPresenter"
+                                          Grid.Row="0"
+                                          IsHitTestVisible="False">
+                            <!-- Margin is on the StackPanel because ContentPresenter clips differently such that moving the reverse margin up won't work -->
+                            <StackPanel Margin="-40,-40,-40,-40"
+                                        Orientation="Horizontal">
+                                <StackPanel x:Name="RatingForegroundStackPanel"
+                                            Margin="40,40,40,40"
+                                            IsHitTestVisible="False"
+                                            Orientation="Horizontal" />
+                            </StackPanel>
+                        </ContentPresenter>
+                    </Grid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+</ResourceDictionary>
```

Comment:
```suggestion
	<Style x:Key="MaterialDefaultRatingControlStyle"
		   TargetType="muxc:RatingControl"
		   BasedOn="{StaticResource MaterialRatingControlStyle}" />
</ResourceDictionary>
```

@nguyeenvy missing the default style (to match the coming changes https://github.com/unoplatform/Uno.Themes/pull/1044/files)